### PR TITLE
Feature/websocket

### PR DESCRIPTION
**Pull request by katherinebernstein**
websockets!

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -217,6 +217,15 @@ dependencies {
...
      testCompile 'com.squareup.retrofit:retrofit-converters:1.9.0'
+     
+    /*
+     * WEBSOCKET DEPENDENCIES
+     */
+      
+      compile "org.springframework:spring-websocket:${project.spring_framework_version}"
+      compile "org.springframework:spring-messaging:${project.spring_framework_version}"
+      compile "org.springframework.integration:spring-integration-websocket:4.2.0.M2"
+      testCompile("junit:junit")
```
Comment (geekbeast) [inline build.gradle]: How is this different from junit above?

File: src/main/java/com/kryptnostic/helper/services/v1/MyHandler.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import org.eclipse.jetty.websocket.api.WebSocketBehavior;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.socket.WebSocketHandler;
+import org.springframework.web.socket.WebSocketSession;
+import org.springframework.web.socket.TextMessage;
+import org.springframework.web.socket.handler.TextWebSocketHandler;
+
+import com.geekbeast.rhizome.configuration.websockets.WebSocketConfig;
+
+public class MyHandler extends TextWebSocketHandler {
```
Comment (geekbeast) [inline src/main/java/com/kryptnostic/helper/services/v1/MyHandler.java]: Is this an example handler?

File: src/main/java/com/kryptnostic/helper/services/v1/HandshakeInterceptor.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import org.springframework.http.server.ServerHttpResponse;
+import org.springframework.web.socket.server.support.DefaultHandshakeHandler;
+import org.springframework.web.socket.server.support.HttpSessionHandshakeInterceptor;
+
+import com.geekbeast.rhizome.pods.RethinkDbPod;
+
+@EnableAsync
+public class HandshakeInterceptor extends HttpSessionHandshakeInterceptor {
+	
+//	@Override
```
Comment (geekbeast) [inline src/main/java/com/kryptnostic/helper/services/v1/HandshakeInterceptor.java]: no commented code :-1:

File: build.gradle (hunk 1 above)
Comment (katherinebernstein) [inline build.gradle]: it's not, and now it's gone! oops

File: src/main/java/com/kryptnostic/helper/services/v1/MyHandler.java (hunk 2 above)
Comment (katherinebernstein) [inline src/main/java/com/kryptnostic/helper/services/v1/MyHandler.java]: it was and then I forgot about it. file is now removed

**Comment by makosblade** [comment]
LGTM

**Pull request merged**